bin/setup

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env ruby
+require 'pathname'
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+
+def install_dependencies
+  puts "== Installing dependencies =="
+  system "gem install bundler --conservative"
+  system "bundle check || bundle install"
+  system "npm install"
+  system "bundle exec rake assets:webpack"
+end
+
+def copy_sample_configuration
+  return if File.exist?("config/database.yml")
+
+  puts "\n== Copying sample configuration files =="
+  puts " Please choose the adapter (press enter to skip):"
+  puts "  PostgreSQL adapter   : 1"
+  puts "  MySQL adapter        : 2"
+  print " (press enter to skip) : "
+
+  adapter_i = Integer(gets.chomp) rescue 0
+
+  adapter = case adapter_i
+    when 1
+      "postgres"
+    when 2
+      "mysql"
+    else
+     ""
+  end
+
+  if adapter.empty?
+    puts "\n Adapter not set! Skipping copying of database.yaml files."
```

Comment:
Maybe I should wrap it in an if statement then ;-) ... No seriously good point. I could also write `puts "", "Adapter ..." to still keep my one-liner.